### PR TITLE
Add prefix setting for C++/Python reader

### PR DESCRIPTION
**Pull request by yush1ga**
### Motivation
https://github.com/apache/incubator-pulsar/pull/1272
Now C++ reader cannot be used when subscription auth mode is prefix.

### Modifications

Add prefix setting for C++ reader.

### Result

C++ reader can be used when subscription auth mode is prefix.

### CODE REVIEW

**Comment by yush1ga** [comment]
retest this please

**Comment by yush1ga** [comment]
retest this please

**Comment by yush1ga** [comment]
retest this please

File: pulsar-client-cpp/python/src/config.cc [hunk 1]
```diff
@@ -142,5 +142,7 @@ void export_config() {
             .def("receiver_queue_size", &ReaderConfiguration::setReceiverQueueSize)
             .def("reader_name", &ReaderConfiguration::getReaderName, return_value_policy<copy_const_reference>())
             .def("reader_name", &ReaderConfiguration::setReaderName)
+            .def("reader_name", &ReaderConfiguration::getSubscriptionRolePrefix, return_value_policy<copy_const_reference>())
+            .def("reader_name", &ReaderConfiguration::setSubscriptionRolePrefix)
```
Comment (merlimat) [inline pulsar-client-cpp/python/src/config.cc]: These 2 lines should have the `"subscription_role_prefix"` and not `"reader_name"`

Comment (yush1ga) [reply pulsar-client-cpp/python/src/config.cc]: Sorry, fixed.

**Pull request merged**